litr-demo/src/main/java/com/linkedin/android/litr/demo/fragment/CameraSizes.kt

```diff
@@ -32,32 +30,18 @@ class SmartSize(width: Int, height: Int) {
 @RequiresApi(Build.VERSION_CODES.LOLLIPOP)
 val SIZE_1080P: SmartSize = SmartSize(1920, 1080)
 
-/** Returns a [SmartSize] object for the given [Display] */
-@RequiresApi(Build.VERSION_CODES.LOLLIPOP)
-fun getDisplaySmartSize(display: Display): SmartSize {
-    val outPoint = Point()
-    display.getRealSize(outPoint)
```

Comment:
This is deprecated. Since it was only there to handle devices with "smaller than 1080p screens" and is only demo app code, I decided to simplify the code and just remove it.